### ISSUE TITLE
Transparency portal broken

### ISSUE THREAD

**Issue by john-light** [aragon/multisig-transparency#13]
![screenshot_2019-02-25 aragon funds](https://user-images.githubusercontent.com/9424721/53386565-a8d49780-3937-11e9-91fb-e61ca8e51263.png)

**Comment by bpierre** [comment]
This happens when there is no injected Ethereum provider and our own can’t be reached: https://github.com/aragon/multisig-transparency/blob/master/services/Wallet.js#L32

As an easy fix, we should probably keep trying instead of showing the error.

@john-light Could you confirm that it only happens sometimes?

**Comment by john-light** [comment]
@bpierre yes the error is happening consistently in Firefox, even with all extensions deactivated. In Chromium the page works.

**Comment by luisivan** [comment]
It's working for me on latest Firefox on Arch Linux, weird

**Comment by john-light** [comment]
After some debugging with @delfipolito  we figured out the problem was Frame. When I deactivated Frame, the error went away. The web page seems to have some weird interaction with Frame.

**Comment by john-light** [comment]
This issue now shows up whether or not the user has Frame or MetaMask enabled.
